CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Code freeze date: YYYY-MM-DD
 
 ### Added
 
+- Add coastal and riverine flood data from Aqueduct [#108](https://github.com/CLIMADA-project/climada_petals/pull/108) [#100](https://github.com/CLIMADA-project/climada_petals/pull/100)
```

Comment:
would be nice to list the methods that have been added, removed and changed.
As I see it the changelog is something a developer would read when their code breaks after an update. They should be immediately pointed to the cause of breaking and a way to fix it.